rust/src/nasl/utils/context.rs

```diff
@@ -398,6 +399,15 @@ impl<'a> Context<'a> {
         &self.target
     }
 
+    /// Get the target host
+    pub fn target_ip(&self) -> Result<IpAddr, AddrParseError> {
+        match self.target() {
+            x if !x.is_empty() => x.to_string(),
+            _ => "127.0.0.1".to_string(),
```

Comment:
q: do we want to use localhost if there is no target provided? Or shouldn't we enforce that a target is set when we start to run?

Reply:
I kept this as in the original code, but I agree with you and would much prefer doing any kind of logic like this once when we start the run and then never touch it again. I think @Kraemii wanted to change this behavior originally, but I don't remember the reason why this was put on hold.

Reply:
The reason this was put on hold, was that I wanted to enforce an IP address for the target provided within the context. It was said to discuss this first, as it is may be wanted to also have a host name as a target

Reply:
Opened jira issue for this